recipe/meta.yaml

```
@@ -12,17 +12,18 @@ source:
 
 build:
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
-  number: 0
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 1
 
 requirements:
   host:
-    - pip >=21.3
-    - python >=3.8
+    - pip
+    - python
     - flit
+    - wheel
   run:
-    - numpy >=1.20
-    - python >=3.8
+    - numpy {{ numpy }}
```

```suggestion
    - numpy >=1.21
```
 The `{{ numpy }}` notation is for `host` not for `run`

---

how about `{{ pin_compatible('numpy') }}`? `cbc.yaml` supports only `numpy >= 1.21`

---

Yes, it's possible